mkosi/backend.py

```diff
@@ -415,7 +415,8 @@ class MkosiArgs:
     repositories: List[str]
     use_host_repositories: bool
     repos_dir: Optional[str]
-    architecture: Optional[str]
+    architecture: str
+    host_architecture: str
```

Comment:
I don't think we should store this in `host_architecture`. We can add a more clearly named function to get the current host architecture and just use that everywhere.

Reply:
That'll do the trick